webapp/stores/browser_store.jsx

```diff
@@ -160,10 +161,14 @@ class BrowserStoreClass {
         if (landingPageSeen) {
             this.setLandingPageSeen(landingPageSeen);
         }
+
+        if (selectedTeams) {
+            this.setItem('selected_teams', selectedTeams);
+        }
     }
 
     isLocalStorageSupported() {
-        if (this.checkedLocalStorageSupported !== '') {
+        if (this.checkedLocalStorageSupported !== undefined) { // eslint-disable-line no-undefined
```

Comment:
It has to be something not true/false since we use that to tell if we can use local storage or not. The old version was the empty string (not checked), true (checked and could use local storage), or false (checked, but can't use local storage). I just went with undefined so we weren't mixing types so horribly